### PR TITLE
【Hackathon 9th No.72】supplementary unit test for worker/model_runner

### PR DESCRIPTION
**Pull request by Echo-Nie**
### Test Content

* Verify that `load_model()` successfully returns the mock model and that it can be correctly retrieved via `get_model()`.

* Call `execute_model()` with `batch_size=4`, `prompt_tokens=6`, and `decode_tokens=8`.
* Check the returned `ModelRunnerOutput`:

  1. `generated_ids` and `logits` have the correct batch dimension.
  2. Each sequence contains the expected number of decode tokens.

* Call `profile_run()` to return mocked profile information, including the key field `memory`.

### Result
```
/opt/conda/envs/python35-paddle120-env/lib/python3.10/site-packages/paddle/utils/cpp_extension/extension_utils.py:717: UserWarning: No ccache found. Please be aware that recompiling all source files may be required. You can download and install ccache from: https://github.com/ccache/ccache/blob/master/doc/INSTALL.md
  warnings.warn(warning_message)
/opt/conda/envs/python35-paddle120-env/lib/python3.10/site-packages/_distutils_hack/__init__.py:30: UserWarning: Setuptools is replacing distutils. Support for replacing an already imported distutils is deprecated. In the future, this condition will fail. Register concerns at https://github.com/pypa/setuptools/issues/new?template=distutils-deprecation.yml
  warnings.warn(
...
----------------------------------------------------------------------
Ran 3 tests in 0.005s

OK
```

### CODE REVIEW

File: tests/worker/model_runner.py
Comment (gongshaotian) [inline tests/worker/model_runner.py:1]: 建议参考下gpu_model_runner的 dummy run 方法，dummy run 会构造特定长度的输入

**Comment by luotao1** [comment]
@Echo-Nie Any update?

**Comment by Echo-Nie** [comment]
之前没来得及维护，该PR暂时关闭
